### PR TITLE
Fixed Bahamut Shark (Anime) and Double-RUM Utopia Force - Now they work with the 2020 Rules card.

### PR DESCRIPTION
**Pull request by Astral-1**
Fixed Bahamut Shark (Anime). Now it works properly with the 2020 Rules card.

8/03/2020: Fixed Double-RUM Utopia Force, now it works properly with the 2020 Rules card.

Added new Card: Neo-Spacian Aqua Dolphin (Anime), unlike the TCG version it allows usage of its effect more than once per turn

### CODE REVIEW

File: script/c511000685.lua [hunk 1]
```diff
@@ -1,71 +1,71 @@
---ＷＲＵＭ－ホープ・フォース
```
Comment (kevinlul) [inline script/c511000685.lua]: Did you change the line endings in this file?

Comment (Astral-1) [reply script/c511000685.lua]: Nope, i didn't. I do not know why that happened

**Comment by kevinlul** [comment]
Please use a separate pull request for the unrelated change.

**Comment by Astral-1** [comment]
Done.

**Pull request merged**